### DISCUSSION TITLE
Selectable Custom Loading Screen and Progress Bar

### DISCUSSION THREAD

**Discussion by ZeroAlvein** [4ian/GDevelop#4707] [Enhancement ideas (and their technical discussions)]
I see users come to discord and ask how to disable that loading screen with gdevelop logo when game starts

Well i do in fact want people to know i made my game with Gdevelop so they be more willing to try it out themselves
So i don’t want to remove that screen
What i would wish for is
- Allow users to submit their concepts for it
- Same for progress bar
- Make it so we can select it one way or another in gdevelop (even from asset store)
And i believe that would part way solve the issue
Since ppl could design other logo variations same as progress bars which users would be more willing to use/choose
For example if im making pixel art game i would wish to have that logo being kinda low res/pixelated same with progress bar
And i would be more willing to use that one over what we have now instead of looking for a way to remove it
Even in game properties option to select from few would be a cool option to have

**Comment by 4ian** [comment]
Agree with what you said. We'll have something in this sense in the next weeks.
Note that there are already some options you can enable in the properties of the game - not sure if you saw that?

But we'll start to giving different choices in the future :)
